vowpalwabbit/log_multi.cc

```diff
@@ -303,7 +303,8 @@ void predict(log_multi& b, single_learner& base, example& ec)
 {
   MULTICLASS::label_t mc = ec.l.multi;
 
-  ec.l.simple = {FLT_MAX, 0.f, 0.f};
+  ec.l.simple = {FLT_MAX, VW::UNUSED_1, VW::UNUSED_0};
```

Comment:
if they are unused why does the naming convention matter? what does _1 and _0 represent?

Reply:
The constants are defined as 1 and 0